<?xml version="1.0"?>
<!-- 
hokuyo_ust_10lx.launch

Created by: Robson RogÃ©rio Dutra Pereira on 01.Out.2022
Last Modified: rrdpereira

Description: This is the launch for the Hokuyo UST 10 LX.
-->
<launch>
  <node name="urg_node10lx" pkg="urg_node" type="urg_node" output="screen">
    <param name="ip_address" value="192.168.1.15"/>
    <param name="serial_port" value="/dev/ttyACM0"/>
    <param name="serial_baud" value="115200"/>
    <param name="frame_id" value="laser"/>
    <param name="calibrate_time" value="true"/>
    <param name="publish_intensity" value="false"/>
    <param name="publish_multiecho" value="false"/>
    <param name="angle_min" value="-1.5707963"/>
    <param name="angle_max" value="1.5707963"/>
  </node>
</launch>